packages/apex-node/src/tests/codeCoverage.ts

```diff
@@ -174,7 +174,9 @@ export class CodeCoverage {
     const queries = this.createQueries(selectQuery, idSet);
 
     const queryPromises = queries.map(query => {
-      return this.connection.tooling.autoFetchQuery(query) as Promise<T>;
+      return (this.connection.tooling.query(query, {
+        autoFetch: true
+      }) as unknown) as T;
```

Comment:
@gbockus-sf I don't think this will work. Query returns `QueryResult<T>` and your function is returning a `T[]`
``` suggestion
      return this.connection.tooling.query<T>(query, {
        autoFetch: true
      }).then(queryResult => queryResult.records);
```

Reply:
Or you choose to break query/then into two statements.

Reply:
Good catch @peternhale .  I'll hit you up on slack as I'd like to understand how the two type systems work together

Reply:
Updated.  Thanks.